react-graphql-apollo-tutorial.md

```diff
@@ -1345,9 +1341,9 @@ const updateAddStar = (
 };
 {{< /highlight >}}
 
-Note: An alternative way would have been to pass this `id` of the repository to the `updateAddStar()` function, which would be a higher-order function then, in the Mutation component's render prop child function. After all, you already have access to the identifier of the repository in the Repository component.
+You could have passed the `id` of the repository to the `updateAddStar()` function, which was a higher-order function in the Mutation component's render prop child function.  You already have access to the repository's identifier in the Repository component.
 
-Now comes the most exciting part of this section. You can use the Apollo Client to read data from the cache but also to write data to the cache. The goal is to read the starred repository from the cache (hence the id), increment its count of stargazers by one, and write the updated repository back to the cache. But how to get the repository by its `id` from the cache in the first place? That's why you have extracted the repository fragment before. You can use it along with the repository identifier to retrieve the actual repository from Apollo Client's cache without querying all the data with a naive query implementation.
+Now comes the most exciting part of this section. You can use the Apollo Client to read data from the cache,a but also to write data to it. The goal is to read the starred repository from the cache (the id), increment its stargazers count of by one, and write the updated repository back to the cache.  You got repository by its `id` from the cache by extracting the repository fragment. You can use it along with the repository identifier to retrieve the actual repository from Apollo Client's cache without querying all the data with a naive query implementation.
```

Comment:
"from the cache,a but also to write data to it" to "from the cache, but also to write data to it"

Reply:
"The goal is to read the starred repository from the cache (the id)" to "The goal is to read the starred repository from the cache (therefore the id)"

Reply:
Alway avoid putting whole phrases in parenthesis.  If you need to give additional info, just start a new sentence or sentence fragment.  "...The goal is to read the starred repository from the cache, which is why we need the `id` to increment its stargazers count by one and write the updated repository back to the cache."